### PR TITLE
Avoid intercom to allow message when we are closed

### PR DESCRIPTION
**Pull request by WengerK**

### CODE REVIEW

File: drupal/sites/all/themes/antistatique/templates/html.tpl.php [hunk 1]
```diff
@@ -112,12 +112,15 @@
...
 
   <!-- intercom -->
-  <script>
-    window.intercomSettings = {
-      app_id: "l40asf6j"
-    };
-  </script>
-  <script>(function(){var w=window;var ic=w.Intercom;if(typeof ic==="function"){ic('reattach_activator');ic('update',intercomSettings);}else{var d=document;var i=function(){i.c(arguments)};i.q=[];i.c=function(args){i.q.push(args)};w.Intercom=i;function l(){var s=d.createElement('script');s.type='text/javascript';s.async=true;s.src='https://widget.intercom.io/widget/l40asf6j';var x=d.getElementsByTagName('script')[0];x.parentNode.insertBefore(s,x);}if(w.attachEvent){w.attachEvent('onload',l);}else{w.addEventListener('load',l,false);}}})()</script>
+  <!-- Avoid intercom to allow message when we are closed -->
+  <?php if ( (date('N') < 6) && date('H') > 8 && date('H') < 18 ): ?>
```
Comment (gido) [inline drupal/sites/all/themes/antistatique/templates/html.tpl.php]: what about the page cache ?

Comment (gido) [inline drupal/sites/all/themes/antistatique/templates/html.tpl.php]: do this in javascript if the Intercom API offer it ?

**Comment by Sudei** [comment]
Done with JS instead to avoid cache issues.

File: drupal/sites/all/themes/antistatique/templates/html.tpl.php [hunk 2]
```diff
@@ -119,6 +119,28 @@
...
+        $(document).ready(function(){
+            var d = new Date();
+            var hours = isDST(d) ? d.getUTCHours() + 2 : d.getUTCHours() + 1;
+            var day = d.getUTCDay();
+            var isWeekDay = day > 0 && day < 6;
+            if (isWeekDay && hours >= 8 && hours < 18) {
+                Intercom('shutdown');
+            }
+
+            // display shop open/closed sign
```
Comment (gido) [inline drupal/sites/all/themes/antistatique/templates/html.tpl.php]: comment are not related to the code

**Pull request merged**